src/components/Settings/ColumnSettings/index.jsx

```diff
@@ -23,7 +23,7 @@ const ColumnSettings = () => {
     <div>
       <div className={columnEditCont}>
         {data
-          ? data.programs[0].columns.map(column => (
+          ? data.programs[0].statuses.map(column => (
```

Comment:
This conditional may need to be tweaked as it's only checking that data exists, but not if programs contains any information, for example

```
data = { programs: null }
data = {  programs: undefined }
data = { something_else: value }
```

will all pass true on `if( data )` but will return `Cannot read map of undefined` or `__ has no property map`

Reply:
Also noticing that you're directly referencing index 0 so

`data = {  programs: [] }` will also throw an error will trying to access statuses